Doppler.BillingUser/Controllers/BillingController.cs

```diff
@@ -0,0 +1,39 @@
+using Doppler.BillingUser.DopplerSecurity;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Doppler.BillingUser.Controllers
+{
+    [Authorize]
+    [ApiController]
+    public class BillingController
+    {
+        [Authorize(Policies.OWN_RESOURCE_OR_SUPERUSER)]
+        [HttpGet("/accounts/{accountname}/currentbillinginformation")]
```

Comment:
```suggestion
        [HttpGet("/accounts/{accountname}/billinginformation/current")]
```

Reply:
Or simply:

```suggestion
        [HttpGet("/accounts/{accountname}/billinginformation")]
```

I mean, it is the same resource that you are updating with the `PUT`, it should have the same URL.

BTW, what do you think about using dash as the word separator?

```suggestion
        [HttpGet("/accounts/{accountname}/billing-information")]
```

Reply:
You're right about the URL. Thanks!